doc/rados/operations/balancer.rst

```diff
@@ -28,6 +28,9 @@ This will use the ``crush-compat`` mode, which is backward compatible
 with older clients, and will make small changes to the data
 distribution over time to ensure that OSDs are equally utilized.
 
+The automatic balancing can be limited to specific pools with::
```

Comment:
I wonder if we can switch to use the balancer CLIs to do the config so we can backport this change into luminous, where these mgr configs can not be changed online (I guess?)...

Reply:
In luminous they can be changed with ceph config-key set ...

Reply:
yeah,  but IIUC in luminous there is no way for the config-key settings to take effect immediately unless we reboot the mgr process?

Reply:
That's possibly true. ceph mgr fail <active one> also does the trick.
What would you propose exactly to improve this fix?

Reply:
This setting might benefit more generally from having a full-blown setter command in the module -- it could validate the syntax of the list (e.g. make sure user is using "," instead of ";"), and also validate the existence of the named pools.  To make it all super-tight you could also handle osdmap notifications and use them to cull any pools that no longer exist.

Reply:
OK I'll preparing something treating pools like we set mode or active. Thanks for the suggestion!

Reply:
> OK I'll preparing something treating pools like we set mode or active.

Yeah, that sounds better.